api/app/services/solution-service.js

```diff
@@ -21,6 +21,27 @@ function areStringListEquivalent (listA, listB) {
   return result;
 };
 
+function removeAccentsSpacesUppercase (s) {
```

Comment:
renommer la variable `s` en `accentuatedString` (parce qu'une variable ne devrait s'appeler par une lettre, sauf peut-être `i`dans une boucle for, et encore)